docker-compose.example.yml

```diff
@@ -0,0 +1,68 @@
+version: "3"
+services:
+  # The postgres server
+  database:
+    # Pulling the image in the latest tag is a disaster.
+    image: postgres:13-alpine
+    environment: 
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgres
+      - POSTGRE_DB=postgres
+    volumes:
+    - dbdata:/var/lib/postgresql/data
+  
```

Comment:
Try adding this:
```yaml
ports:
  - 5432:5432
```

It will fix the `(DBConnection.ConnectionError) connection not available and request was dropped (...)` error mentioned on https://github.com/benawad/dogehouse/pull/118#issuecomment-778812723.